lib/gateways/unique_connections.rb

```diff
@@ -5,9 +5,14 @@ def initialize(ips:)
     end
 
     def unique_user_count(date_range: nil)
-      return Session.where(siteIP: organisation_ips).where('start >= ?', date_range).where(success: true).distinct.count(:username) if date_range.present?
+      query = Session
+        .where(success: true)
+        .distinct
+        .where(siteIP: organisation_ips)
 
-      Session.where(siteIP: organisation_ips).where(success: true).distinct.count(:username)
+      query = query.where('start >= ?', date_range) if date_range.present?
```

Comment:
When do we use a different date range here?
Realise you didn't introduce this functionality but wondering where it is used.

Reply:
We don't, as far as I know - I guess it was built prioritising flexibility over YAGNI?

I would have removed it as part of this PR, but I thought part of the PR feedback might have been "we should really do this for a week instead of 24 hours"!

I'd be happy to remove it, as I haven't gotten that feedback.

Reply:
Interesting, I'd be tempted to remove it.  Less code to think about